speedcord/ext/typing/context.pyi

```diff
@@ -35,8 +35,7 @@ class MessageContext:
     application: Optional[Dict[str, Any]]
     message_reference: Optional[Dict[str, int]]
     flags: Optional[int]
-    ...
```

Comment:
Maybe add client to messagecontext as well? The typing doesn't seem to inherit from BaseContext

Reply:
You're right, thought the typing would inherit naturally - well explicitly adding them does works, so I've done that.